requirements.txt

```diff
@@ -19,3 +19,4 @@ pytz
 pytest
 tzlocal
 mock
+urllib3<1.25,>=1.21.1
```

Comment:
Please set this to "urllib3=1.22" so it can be consistent with the other pull request and the other asperathos modules

Reply:
Done